citools/dump_failures.py

```diff
@@ -19,6 +19,14 @@
 RESULTS: Dict[str, Dict[str, Dict[str, List[int]]]] = defaultdict(
     lambda: defaultdict(lambda: defaultdict(lambda: [0, 0]))
 )
+SKIPPED_MANIFESTS = []
+
+
+def red_file_ini():
+    file_config = configparser.configparser()
+    file_config.read("dom/canvas/test/mochitest.ini")
+    for test in file_config.sections():
+        SKIPPED_MANIFESTS.append(test)
```

Comment:
great start on this.  I think you went too far here- it is ok a great opportunity for me to explain manifest vs test.  We have hundreds of manifest files (.ini) in the mozilla-central source tree.  Each manifest file contains a set of tests (between 1 and 400+, most contain 20-60).  For SKIPPED_MANIFESTS we want to skip manifests that have special cases.  We don't care as much about the individual tests.  While you did a great job of reading the manifest and adding each test to the SKIPPED_MANIFEST, that isn't needed here, just a list of .ini files.

Reply:
Ok Thanks, now I see it and understand it better. "mochitest.ini" is the manifest file, which I must add to the list.

Reply:
Hello. I have implementd this piece of code
"def update_skipped_manifest():
    path_list = pathlib.Path("dom/canvas/test").glob("**/*.ini")
    for skip_manifest in path_list:
        if skip_manifest.name == "mochitest.ini":
            SKIPPED_MANIFESTS.append(skip_manifest)"
This allows me to find all the .ini files that are in that directory, in this case that path is from the mozilla-central source tree, but in this project (mozci-tools) my sistem couldn't access it. What else should I take into account to go through this directory and not generate failures?. Anyway I executing the test, but isort failed. Could this failure be due to the above mentioned?.